defaults/main.yml

```diff
@@ -20,3 +20,4 @@ node_exporter_enabled_collectors:
 #      ignored-fs-types: "^(sys|proc|auto)fs$"
 
 node_exporter_disabled_collectors: []
+node_exporter_private_tmp: true
```

Comment:
I would prefer if this wasn't conditional as we intend to allow permissions to run most node_exporter collectors without tinkering with permissions.

Could you hardcode it?